defaults/default-prefs.toml

```diff
@@ -69,6 +69,17 @@ path = ".bacon-locations"
 line_format = "{kind} {path}:{line}:{column} {message}"
 
 
+# If you want some job to emit a bip on success or on failure,
```

Comment:
I don't think "bip" is english. Not familiar with it and can't find it on google. Suspect you wanted "beep".